### PR TITLE
fix bandwidth calculation

### PR DESCRIPTION
**Pull request by wenkaidu**
Fix bandwidth calculation also build error due to missing header file

### CODE REVIEW

**Comment by wenkaidu** [comment]
Any reason for using (num_gpus/(num_gpus-1))? nccl-tests doesn't use any factor.

File: tests/performance/rcclAllReduce.cpp [hunk 1]
```diff
@@ -86,7 +86,7 @@ void DoAllReduce(std::vector<int>& device_list, std::vector<hipStream_t>& device
             HIPCHECK(hipStreamSynchronize(device_streams[i]));
         }
         mark.done();
-        mark.bw(buff_size*knum_iter*num_gpus);
+        mark.bw(buff_size*knum_iter);
```
[the remark this answers is not in the document]
Comment (adityaatluri) [reply tests/performance/rcclAllReduce.cpp]: For allreduce, the effective bandwidth is `(num_gpus)/(num_gpus - 1)`

**Comment by wenkaidu** [comment]
@jeffdaily 
Hi Jeff, can you comment on bandwidth calculation?
I think algorithm bandwidth should be just data_size/time. Bus bandwidth for allreduce should be 2*(nranks - 1)/nranks.

**Comment by jeffdaily** [comment]
I agree with @wenkaidu 's calculations.  But it looks like there are now conflicts with master that need fixing.  @wenkaidu you might need to rebase your PR against current master and resubmit a new PR.  But in any case, I agree with @wenkaidu 's changes.

**Comment by wenkaidu** [comment]
Will push new commit.

**Pull request closed without merging**